### ISSUE TITLE
Convince chromium that JPEG XL has an objectively high value proposition

### ISSUE THREAD

**Issue by LifeIsStrange** [libjxl/libjxl#1854]
https://bugs.chromium.org/p/chromium/issues/detail?id=1178058
Please @jonsneyers do something, talk to them..
You can't let one overzealous aGiLe chromium manager ruin the web and incur an insanely big negative ecological and and user experience impact.

**Comment by LifeIsStrange** [comment]
@deymo

**Comment by wischi-chr** [comment]
Not really an libjxl issue per se, but you could also talk to them. Leave a comment on the chromium issue tracker with the reasons why you think it's not a good decision. Star the chromium issue if it's important to you and tell others to do the same.

**Comment by LifeIsStrange** [comment]
Yes already did however they don't seem to listen to the worldwide community nor experts nor even to basic logic. Chromium went rogue. I read a HN comment showing that the "manager" behing the decision (via linkedin profile) had a proeminent role in webp, I see this as a personal/ego conflict of interest. However it seems there is some internal resistance withing chromium see e.g. the recent commit of firshcing that sped up JPEGL XL decoding by 3.
Beyond the chromium manager changing his mind or put to enough pressure by other employees, the only hope I see for the web would be:
Microsoft being relevant and saying they will 'maintain' JPEG XL in chromium tree (AKA update the dependency version once in a while..)
or Collabora,
or Safari or firefox making a move and enabling JPEG XL by default. This would bring them excellent marketing and real world impact therefore increasing their relevance, and eventually, their marketshares.

**Comment by jonnyawsom3** [comment]
I'd say out of scope and already tried to no avail. Hopefully jxl-rs can swing things in our favour.

**Comment by MasterInQuestion** [comment]
Note that for browsers security would be of extraordinary consideration.
    Alike the reasoning:
[ https://github.com/richtr/NoSleep.js/issues/157#issuecomment-1529149759](https://github.com/richtr/NoSleep.js/issues/157#issuecomment-1529149759)
    "Browsers for security considerations may choose to only support a limited set of medias to reduce attack surface."

    While in my limited testing:
    JXL doesn't appear superior enough comparing to WebP Lossless.
    And probably currently buggy in some aspects.
    .
    Also probably it's still missing some core aspects, of the ideal codec design:
    [ https://github.com/libjxl/libjxl/issues/3921#issuecomment-2513471925](https://github.com/libjxl/libjxl/issues/3921#issuecomment-2513471925)

**Comment by jonnyawsom3** [comment]
We got there in the end https://groups.google.com/a/chromium.org/g/blink-dev/c/WjCKcBw219k/m/NmOyvMCCBAAJ?pli=1
